src/cli/crd/generate.ts

```diff
@@ -0,0 +1,316 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The Pepr Authors
+
+import { Command } from "commander";
+import fs from "fs";
+import path from "path";
+import { stringify as toYAML } from "yaml";
```

Comment:
That seems like an odd alias for a function. I wonder why it wasn't called `toYAML` in the first place...

Reply:
im not sure, copied it from another section of the code and was looking like that. definitely strange

Reply:
see #2068